docset/winserver2022-ps/addsdeployment/Test-ADDSForestInstallation.md

```diff
@@ -11,52 +11,75 @@ title: Test-ADDSForestInstallation
 # Test-ADDSForestInstallation
 
 ## SYNOPSIS
+
 Runs the prerequisites for installing a new forest in Active Directory.
 
 ## SYNTAX
 
 ```
-Test-ADDSForestInstallation -DomainName <String> [-SafeModeAdministratorPassword <SecureString>]
- [-CreateDnsDelegation] [-DatabasePath <String>] [-DnsDelegationCredential <PSCredential>] [-NoDnsOnNetwork]
- [-DomainMode <DomainMode>] [-DomainNetbiosName <String>] [-ForestMode <ForestMode>] [-InstallDns]
- [-LogPath <String>] [-NoRebootOnCompletion] [-SkipAutoConfigureDns] [-SysvolPath <String>] [-Force]
- [<CommonParameters>]
+Test-ADDSForestInstallation -DomainName <String>
+ [-SafeModeAdministratorPassword <SecureString>] [-CreateDnsDelegation]
+ [-DatabasePath <String>] [-DnsDelegationCredential <PSCredential>] [-NoDnsOnNetwork]
+ [-DomainMode <DomainMode>] [-DomainNetbiosName <String>] [-ForestMode <ForestMode>]
+ [-InstallDns] [-LogPath <String>] [-NoRebootOnCompletion] [-SkipAutoConfigureDns]
+ [-SysvolPath <String>] [-Force] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Test-ADDSForestInstallation** cmdlet runs those prerequisite checks which would be performed if you were to use the Install-ADDSForest cmdlet to install a new forest in Active Directory.
-It differs from using the *WhatIf* parameter with the **Install-ADDSForest** cmdlet in that instead of summarizing the changes that would occur during the installation process, this cmdlet actually tests whether those changes are possible given the current environment.
 
-For more information on the scope of these prerequisite checks that the ADDSDeployment module performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in [AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
+The `Test-ADDSForestInstallation` cmdlet runs those prerequisite checks which would be performed if
+you were to use the Install-ADDSForest cmdlet to install a new forest in Active Directory. It
+differs from using the `-WhatIf` parameter with the `Install-ADDSForest` cmdlet in that instead of
+summarizing the changes that would occur during the installation process, this cmdlet actually tests
+whether those changes are possible given the current environment.
+
+For more information on the scope of these prerequisite checks that the ADDSDeployment module
+performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in
+[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```

Comment:
```suggestion
The `Test-ADDSForestInstallation` cmdlet runs those prerequisite checks which would be performed if
you were to use the `Install-ADDSForest` cmdlet to install a new forest in Active Directory. It
differs from using the **WhatIf** parameter with the `Install-ADDSForest` cmdlet in that instead of
summarizing the changes that would occur during the installation process, this cmdlet actually
tests whether those changes are possible given the current environment.

For more information on the scope of these prerequisite checks that the **ADDSDeployment** module
performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in
[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```